Rakefile

```diff
@@ -82,6 +82,7 @@ task(:commits) do
         "\t\t(docs)(DOCUMENT-<digits>)\n" \
         "\t\t(maint)\n" \
         "\t\t(packaging)\n" \
+        "\t\t(l10n)\n" \
```

Comment:
Not a blocker, but since this instance is in the documentation section, do we want to have this be `L10n` to match what actually gets committed, and to encourage any manual commits to match that pattern?